src/modernizer/violations.xml

```diff
@@ -67,6 +67,12 @@
         <comment>Prefer new Configuration(false)</comment>
     </violation>
 
+    <violation>
+        <name>org/joda/time/DateTimeZone.toTimeZone:()Ljava/util/TimeZone;</name>
+        <version>1.8</version>
+        <comment>Avoid DateTimeZone.toTimeZone as it returns GMT for a zone not supported by the JVM. Use TimeZone.getTimeZone(ZoneId.of(dtz.getId())) instead.</comment>
```

Comment:
> as it returns GMT for a zone not supported by the JVM

wow!

Reply:
I will likewise block use of `TimeZone.forTimeZone(String)`.